subsys/llext/llext_mem.c

```diff
@@ -77,7 +79,7 @@ static int llext_copy_region(struct llext_loader *ldr, struct llext *ext,
 	 * program-accessible data (not to string tables, for example).
 	 */
 	if (region->sh_flags & SHF_ALLOC) {
-		if (IS_ENABLED(CONFIG_ARM_MPU)) {
+		if (IS_ENABLED(CONFIG_ARM_MPU) | IS_ENABLED(CONFIG_ARC_MPU)) {
```

Comment:
mathematically it works and will have the same result, but logically you probably wanted `||` here

Reply:
Oops! Fixed